fcrepo-api-x-integration/pom.xml

```diff
@@ -28,11 +29,12 @@
         </executions>
         <configuration>
```

Comment:
@DiegoPino  Yes, we could try setting fork count to 0, though if I recall doing so in the past caused additional failures on Elliot's mac (but only on the mac).  It may be worth giving it a quick try before next week to see what happens.